Bottom: 03b18e8aab4a45da539dc081626c2faf86362a91
Top:    f69fe3cc5b47c5cfb2b0286bfb072f9a3d29b1f9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 17:32:25 +0100

Refresh of hid-gtk-turn-on-defines-to-deb

---

diff --git a/configure.ac b/configure.ac
index d2e2eeb..1150f22 100644
--- a/configure.ac
+++ b/configure.ac
@@ -771,7 +771,7 @@ $GTKGLEXT_PKG_ERRORS])]
 	fi
 
 	GTK_CFLAGS="$GTK_CFLAGS -DGSEAL_ENABLE"
-#	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED -DGSEAL_ENABLE"
+	GTK_CFLAGS="$GTK_CFLAGS -DGDK_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
 
 	;;
